app/frontend/components/common/footer.js

```diff
@@ -23,19 +33,78 @@ const Footer = () =>
           'View on Github'
         )
       ),
-      '/',
+      ' /',
       h(
         'small',
         {class: 'contact-link'},
-        h('a', {href: 'mailto:cardanolite@vacuumlabs.com'}, 'Contact us')
+        h('a', {href: 'mailto:cardanolite@vacuumlabs.com', target: '_blank'}, 'Contact us')
       ),
       '/',
       h(
         'small',
         {class: 'contact-link'},
-        h('a', {href: 'https://twitter.com/hashtag/cardanolite'}, '#cardanolite')
+        h('a', {href: 'https://twitter.com/hashtag/cardanolite', target: '_blank'}, '#cardanolite')
+      )
+    ),
+    (donateBtcUrl || donateEthUrl || donateAdaAddress) &&
```

Comment:
originally I thought that this check would make sense, but now I think that it just bring unnecesary corner cases with displaying the slashes right to the code. We already know what currencies we want. Therefore I would remove the check for the presence of BTC, ADA, ETH donation address for the sake of simplicity.